models/classes/ItemPreviewer.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+namespace oat\taoQtiTest\models;
+
+use oat\taoDelivery\model\RuntimeService;
+use oat\taoItems\model\pack\ItemPack;
+use oat\taoItems\model\pack\Packer;
+use oat\taoQtiItem\model\qti\Service;
+use oat\taoQtiItem\model\QtiJsonItemCompiler;
+use oat\taoQtiTest\models\container\QtiTestDeliveryContainer;
+use oat\taoResultServer\models\classes\ResultServerService;
+use Zend\ServiceManager\ServiceLocatorAwareInterface;
+use Zend\ServiceManager\ServiceLocatorAwareTrait;
+
+class ItemPreviewer implements ServiceLocatorAwareInterface
```

Comment:
I'm not sure this modeling is correct. It may not be  linked to the previewer, I can see other uses cases when we can load the compiled item data.

Reply:
You mean, setting a more generic name?

Reply:
yes if you have some inspiration (I don't). I can see multiple use case where you could want to load the item data alone.